src/pgrepr/src/types.rs

```diff
@@ -118,7 +118,8 @@ impl From<&ScalarType> for Type {
             ScalarType::Bytes => Type::Bytea,
             ScalarType::String => Type::Text,
             ScalarType::Jsonb => Type::Jsonb,
-            ScalarType::Array(_t) => unimplemented!("jamii/array"),
+            // there isn't an obvious way to assign an oid to a parametric type
+            ScalarType::List(_t) => Type::Unknown,
```

Comment:
We can just manually pick one to represent the "list" generic type, just like [2277 represents the generic array type](https://github.com/postgres/postgres/blob/4c04be9b05ad2ec5acd27c3417bf075c13cab134/src/include/catalog/pg_type.dat#L531-L535). 16384 and up are fair game: https://github.com/postgres/postgres/blob/4c04be9b05ad2ec5acd27c3417bf075c13cab134/src/include/access/transam.h#L121-L152

Reply:
Tangent, but can you actually return 2277? I thought pseudotypes are for describing function arguments.

Reply:
Yeah according to this comment Postgres does in some places: https://github.com/postgres/postgres/blob/24e2885ee304cb6a94fdfc25a1a108344ed9f4f7/src/backend/utils/adt/pseudotypes.c#L149-L150

Reply:
Mmmmm, consistent :D